variables.md

```diff
@@ -198,6 +198,7 @@ ZFCP_ADAPTERS | string | | Comma separated list of available ZFCP adapters in th
 LINUXRC_BOOT | boolean | true | To be used only in scenarios where we are booting an installed system from the installer medium (for example, a DVD) with the menu option "Boot Linux System" (not "boot From Hard Disk"). This option uses linuxrc.
 ZYPPER_WHITELISTED_ORPHANS | string | empty | Whitelist expected orphaned packages, do not fail if any are found. Upgrade scenarios are expecting orphans by default. Used by console/orphaned_packages_check.pm
 PUBLIC_CLOUD_CONTAINER_IMAGES_REPO | string | | The Container images repository in CSP
+PUBLIC_CLOUD_TERRAFORM_FILE | string | | Custom terraform file for publiccloud test runs
```

Comment:
looks like you forgot about https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/14730

Reply:
Damn, yes! :sweat_smile: